.changeset/dry-owls-teach.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/toggle': minor
+---
+
+- Exports `getLGToggleTestUtils`, a util to reliably interact with `LG Toggle` in a product test suite. [LG-4035](https://jira.mongodb.org/browse/LG-4035)
```

Comment:
I think we can c/p the docs from README here, ditto the other two packages

Reply:
Maybe not copy, but link since it's pretty long for release notes